src/wui/interactive_base.cc

```diff
@@ -409,6 +409,7 @@ InfoToDraw InteractiveBase::get_info_to_draw(bool show) const {
 
 void InteractiveBase::unset_sel_picture() {
 	set_sel_picture(g_image_cache->get("images/ui_basic/fsel.png"));
+	set_tooltip("");
```

Comment:
set_tooltip() should do as empty string is default anyway.
Would an explicit clear_tooltip() help us?

Reply:
`set_tooltip` doesn't have a default value. `set_tooltip("")` is used in a lot of places, imho it's clear that this usage removes the panel's tooltip